src/mlpack/methods/random_forest/CMakeLists.txt

```diff
@@ -21,3 +21,11 @@ add_julia_binding(random_forest)
 add_go_binding(random_forest)
 add_r_binding(random_forest)
 add_markdown_docs(random_forest "cli;python;julia;go;r" "classification")
+#Adding extra trees binding
```

Comment:
Personally I don't think this comment is necessary---I think it is pretty obvious from the code that we are adding the binding for `extra_trees`. :+1: